projects/driver_controls/src/digital_io.c

```diff
@@ -0,0 +1,121 @@
+#include <stddef.h>
+
+#include "gpio_it.h"
+#include "digital_io.h"
+#include "input_event.h"
+
+// Digital device identifiers
+typedef enum {
+  DIGITAL_IO_POWER_SWITCH = 0,
+  DIGITAL_IO_DIRECTION_SELECTOR,
+  DIGITAL_IO_CRUISE_CONTROL,
+  DIGITAL_IO_CRUISE_CONTROL_INC,
+  DIGITAL_IO_CRUISE_CONTROL_DEC,
+  DIGITAL_IO_TURN_SIGNAL,
+  DIGITAL_IO_HAZARD_LIGHT,
+  DIGITAL_IO_MECHANICAL_BRAKE,
+  NUM_DIGITAL_IO_INPUTS
+} DigitalIODevices;
```

Comment:
Same thing with prefixing - so for example, `DIGITAL_IO_DEVICE_POWER_SWITCH`.

Also, same thing as before about enums not being plural - it doesn't quite sound right if you were to actually use it as a type.